R/fit.R

```diff
@@ -403,6 +427,8 @@ mmrm <- function(formula,
   if (!is.null(fit_msg)) {
     message(paste(fit_msg, collapse = "\n"))
   }
+  fit$call <- match.call()
+  fit$call$formula <- formula
```

Comment:
obtain the formula so that it could be printed, instead of a object.

Reply:
ok